src/components/Histogram/HistogramComponent.tsx

```diff
@@ -204,9 +205,18 @@ export class HistogramComponent extends React.Component<WidgetProps> {
                 if (nearest?.point) {
                     let valueLabel = `${nearest.point.y !== 0.5 ? nearest.point.y : 0}`;
 
-                    const frame = AppStore.Instance.activeFrame;
+                    const frame = this.widgetStore.effectiveFrame;
                     if (frame.unit) {
-                        numberString += ` ${frame.unit}`;
+                        let unit: string;
```

Comment:
We have twice of this calculation when rendering. Perhaps set an input `genProfilerInfo(unit)` or add a computed variable.

Reply:
done